libs/common/src/constants/portal/activity.ts

```diff
@@ -3,9 +3,11 @@ export const PORTAL_ACTIVITY_LABEL = {
   MIN_SUBMISSION: 'Manual inclusion notice submitted to UKEF',
   AIN_SUBMISSION: 'Automatic inclusion notice submitted to UKEF',
   FACILITY_CHANGED_ISSUED: 'Bank facility stage changed',
+  DEAL_CANCELLED: 'Deal cancelled',
 } as const;
 
 export const PORTAL_ACTIVITY_TYPE = {
```

Comment:
is this portal specific or GEF?

Reply:
It's GEF specific - only GEF has activities in portal.

Whilst it seems misleading for the (existing) name to be called `PORTAL_ACTIVITY_TYPE` instead of `PORTAL_GEF_ACTIVITY_TYPE`, equally I think the existing name is more scalable incase Portal activities become non-GEF specific.

What do you think? Don't want to scope creep this PR